### PR TITLE
Allow users to switch to using KituraNIO

### PR DESCRIPTION
**Pull request by mamabusi**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,33 +10,46 @@ matrix:
...
     - os: linux
       dist: trusty
       sudo: required
       env: TESTDB_NAME=testdb_2
     - os: linux
       dist: trusty
       sudo: required
-      env: SWIFT_SNAPSHOT=swift-4.2-DEVELOPMENT-SNAPSHOT-2018-06-19-a
-      env: TESTDB_NAME=testdb_3
+      env: SWIFT_SNAPSHOT=swift-4.2-DEVELOPMENT-SNAPSHOT-2018-06-19-a TESTDB_NAME=testdb_3
```
Comment (djones6) [inline .travis.yml]: This should just be '4.2'

File: .travis.yml [hunk 2]
```diff
@@ -10,33 +10,46 @@ matrix:
...
+    - os: linux
+      dist: trusty
+      sudo: required
+      env: SWIFT_SNAPSHOT=swift-4.2-DEVELOPMENT-SNAPSHOT-2018-06-19-a KITURA_NIO=1 TESTDB_NAME=testdb_4
```
Comment (djones6) [inline .travis.yml]: This should just be '4.2'

File: .travis.yml [hunk 3]
```diff
@@ -10,33 +10,46 @@ matrix:
...
+      sudo: required
+      env: SWIFT_SNAPSHOT=swift-4.2-DEVELOPMENT-SNAPSHOT-2018-06-04-a KITURA_NIO=1 TESTDB_NAME=testdb_8
+    - os: osx
+      osx_image: xcode10
+      sudo: required
+      env: SWIFT_SNAPSHOT=4.2 TESTDB_NAME=testdb_9
+    - os: osx
+      osx_image: xcode10
+      sudo: required
+      env: SWIFT_SNAPSHOT=4.2 KITURA_NIO TESTDB_NAME=testdb_10
```
Comment (djones6) [inline .travis.yml]: This looks like a typo - should be `KITURA_NIO=1`

File: .travis.yml [hunk 4]
```diff
@@ -10,33 +10,46 @@ matrix:
...
     - os: osx
       osx_image: xcode9.4
       sudo: required
-      env: SWIFT_SNAPSHOT=swift-4.2-DEVELOPMENT-SNAPSHOT-2018-06-04-a
-      env: TESTDB_NAME=testdb_6
+      env: SWIFT_SNAPSHOT=swift-4.2-DEVELOPMENT-SNAPSHOT-2018-06-04-a TESTDB_NAME=testdb_7
+    - os: osx
+      osx_image: xcode9.4
+      sudo: required
+      env: SWIFT_SNAPSHOT=swift-4.2-DEVELOPMENT-SNAPSHOT-2018-06-04-a KITURA_NIO=1 TESTDB_NAME=testdb_8
```
Comment (djones6) [inline .travis.yml]: These two are redundant as xcode10 + 4.2 (release build) are included below

**Review by djones6 — CHANGES_REQUESTED** [review]
The .travis.yml needs sorting out...

**Comment by mamabusi** [comment]
@djones6 Thanks for the review. Have addressed the changes requested.

**Review by djones6 — APPROVED** [review]

**Review by ianpartridge — APPROVED** [review]

**Pull request merged**